src/font.rs

```diff
@@ -59,6 +60,18 @@ impl CGFont {
         }
     }
 
+    pub fn create_copy_from_variations(&self, vars: &CFDictionary) -> Result<CGFont, ()> {
+        unsafe {
+            let font_ref = CGFontCreateCopyWithVariations(self.as_ptr(),
+            vars.as_concrete_TypeRef());
```

Comment:
nit: indent this to match the first argument.

Reply:
@jdm Thanks! Done =)